### DISCUSSION TITLE
Any plan to release the v1.0.0 and go out of alpha releases?

### DISCUSSION THREAD

**Discussion by ryukinix** [ory/keto#1309] [Q&A] [answered]
We are using keto in a large corporative system to do granular control access for a complex proprietary feature store ecosystem for @NeowayLabs... Keto is doing fine, we are currently in the 0.9.0-alpha version and waiting for a more consolidated version. But I should ask: is there something planning to go out of the alpha releases?

Keto is a very nice piece of software and we are interested to keep using it, and even contributing if necessary. We would like to see keto achieving some stability at some moment.

**Comment by vinckr** [comment, marked as answer]
Hello @ryukinix 

Not in the next weeks AFAICT, but we will probably share plans at [Ory Summit](https://summit.ory.sh/) this year!

Feel free to also reach out to support@ory.sh if you are looking for commercial support for Ory Keto!

**Reply by beanow-at-crabnebula** [reply]
Since the summit has taken place, was there anything announced about Keto releases?

Interested in both v0 alpha and v1 plans.
(Running into some issues that've already been fixed on `master` since the last March alpha release)

**Reply by ryukinix** [reply]
Any news on that?

**Reply by sebastianvoss** [reply]
We are also interested in this information.

**Reply by aeneasr** [reply]
Hello everyone, we are anticipating to spend more time on Keto next year and ship some long awaited features which would go hand in hand with a 1.0 release
